tripper/backends/collection.py

```diff
@@ -66,31 +66,37 @@ def triples(self, triple: "Triple") -> "Generator[Triple, None, None]":
             else:
                 yield s, p, o
 
-    def add_triples(self, triples: "Sequence[Triple]"):
+    def add_triples(
+        self, triples: "Union[Sequence[Triple], Generator[Triple, None, None]]"
+    ):
         """Add a sequence of triples."""
         for s, p, o in triples:
             # Strange complains by mypy - it assumed that
             # parse_object() is returning a `str` regardless that it
             # has been declared to return the union of `str` and
             # `Literal`.
             v = parse_object(o)
-            o = v if isinstance(v, str) else v.value
+
+            # Possible bug in mypy
+            # parse_object() is declared to return "Union[str, Literal]".
+            # Dispite of that complains mypy about that `v` has no
+            # attribute "value"
+            obj = v if isinstance(v, str) else str(v.value)  # type: ignore
```

Comment:
```suggestion
            obj = v if isinstance(v, str) else str(v.value)
```

I have thested locally and with a draft pull request, I do not get any errors with mypy. Do you have an updated version?

Reply:
The problem came when I updated mypy. I got annoyed with it and changed the return type of utils.parse_object() from `Union[str, Literal]` to `Union[str, Any]` to get rid of these errors...

The source of the problem is that mypy suddenly has got the idea that `Literal` has got a special meaning, since it was added to `typing`. 

It even doesn't understand if I change the return type of utils.parse_object() to `Union[str, tripper.Literal]`. 

Using `Any` in the return works around the problem, so I have removed the comment and the `# type: ignore` comments.